wazimap_ng/datasets/tasks.py

```diff
@@ -74,6 +74,15 @@ def indicator_data_extraction(indicator):
 
     qs = models.DatasetData.objects.filter(**filter_query).exclude(data__Count="").order_by("geography_id")
 
+    if len(groups) > 1:
+        group_values = ["/".join(val) for val in list(set(list(qs.values_list(*groups).distinct())))]
+    elif len(groups) == 1:
+        group_values = [val[0] for val in list(set(list(qs.values_list(*groups).distinct())))]
+    else:
+        group_values = []
+
+    models.ProfileIndicator.objects.filter(indicator=indicator).update(subindicators=group_values)
```

Comment:
Does this only populate the ProfileIndicator during the data extraction task? Doesn't that mean that we will need to
1. Create an indicator
2. Create a profile indicator
3. Re-save the indicator so that the background task will population the profile indicator

Reply:
@adieyal hmm I didn't thought about instance when we create new profile indicator as I was only testing the already created data.

I am adding subindicators to profile indicator on data extraction because dataset may have changed and so we will need to update subindicators as well.

In case of creating new profile indicator we can take data from Indicator data for creating subindicators. I will do this today and update this PR.

One more thing I wanted to do but was unsure about is showing user a pop up saying `Updating indicators resets the ordering of subindicators in profile. Please acknowledge that you have to manually change order of subindicators in profile indicators`
When user saves already created indicator.

Thanks for the review :)